contracts/staking/ServiceStakingBase.sol

```diff
@@ -162,19 +157,22 @@ abstract contract ServiceStakingBase is ERC721TokenReceiver, IErrorsRegistries {
     // Set of currently staking serviceIds
     uint256[] public setServiceIds;
     // Map of approved multisig proxy hashes
-    mapping(bytes32 => bool) mapMultisigHashes;
+    mapping(bytes32 => bool) public mapMultisigHashes;
 
     /// @dev ServiceStakingBase constructor.
     /// @param _stakingParams Service staking parameters.
     /// @param _serviceRegistry ServiceRegistry contract address.
-    /// @param _multisigProxyAddresses Multisig proxy addresses.
-    constructor(StakingParams memory _stakingParams, address _serviceRegistry, address[] memory _multisigProxyAddresses) {
+    /// @param _multisigProxyHashes Multisig proxy hashes.
+    constructor(StakingParams memory _stakingParams, address _serviceRegistry, bytes32[] memory _multisigProxyHashes) {
         // Initial checks
         if (_stakingParams.maxNumServices == 0 || _stakingParams.rewardsPerSecond == 0 ||
-            _stakingParams.minStakingDeposit == 0 || _stakingParams.livenessPeriod == 0 ||
-            _stakingParams.livenessRatio == 0 || _stakingParams.numAgentInstances == 0) {
+            _stakingParams.livenessPeriod == 0 || _stakingParams.livenessRatio == 0 ||
+            _stakingParams.numAgentInstances == 0) {
             revert ZeroValue();
         }
+        if (_stakingParams.minStakingDeposit < 2) {
+            revert LowerThan(_stakingParams.minStakingDeposit, 2);
+        }
```

Comment:
This is done such that somehow the minimum staking deposit is not set to 1 (although very unlikely). If that happens, the NativeToken and Token contracts will be undistinguishable, and ERC20-based service will be possible to stake at the NativeToken staking contract.